companies/513947119.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "iCount",
+    "description": "Everything a business needs and more. Finally, a system that can meet all of your business’s needs from the basic to the most advanced.",
```

Comment:
Consider something less obscure. I.e "Accounting, billing, time management and more for small-medium businesses.